mount/mount_darwin.go

```diff
@@ -0,0 +1,85 @@
+//go:build darwin
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package mount
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+
+	exec "golang.org/x/sys/execabs"
```

Comment:
I think this pkgs functionality was added into os/exec in go 1.19 (we build with 1.20.x) so I'd just use os/exec directly.

Reply:
Done

Reply:
But... See https://github.com/containerd/containerd/pull/7647#issuecomment-1309537983